cmd/crowdsec/main.go

```diff
@@ -251,6 +251,10 @@ func LoadConfig(cConfig *csconfig.Config) error {
 		log.Infof("single file mode : log_media=%s daemonize=%t", cConfig.Common.LogMedia, cConfig.Common.Daemonize)
 	}
 
+	if !flags.TestMode && cConfig.Common.PidDir != "" {
```

Comment:
why the specific check on testmode ?

Reply:
For some reason I though it would create duplicate logs during systemctl re(load/start). Removed the check on test mode in latest commit